test/unified_format.py

```diff
@@ -1026,6 +1026,10 @@ def maybe_skip_test(self, spec):
         if "timeoutMS applied to entire download" in spec["description"]:
             self.skipTest("PyMongo's open_download_stream does not cap the stream's lifetime")
 
+        if "unpin after TransientTransactionError error on abort" in spec["description"]:
+            if os.environ.get("MONGODB_VERSION", "") == "latest":
```

Comment:
Can we use client_context.version instead? I don't set MONGODB_VERSION locally when testing.

Reply:
Done